### DISCUSSION TITLE
Installation using conda

### DISCUSSION THREAD

**Discussion by JingChen-Thu** [TomoATT/TomoATT#16] [Q&A]
Prof. Xueyuan Huang, one TomoATT user, encountered difficulties in installing TomoATT using conda. 

He used the following commands to install TomoATT:
conda create -n test
conda activate test
conda install -c conda-forge tomoatt pytomoatt

However, an error occurred when input TOMOATT:

![12c269099a24d489c9199d74b6687aa](https://github.com/user-attachments/assets/17e26b00-33cf-4d53-ba20-cf16b0d278fe)

Do you guys have any idea about this issue? If any additional information is required, please feel free to tell me. I will ask Prof. Huang for necessary information.

**Comment by JingChen-Thu** [comment]
Another question is that, most new users, which use conda to install TomoATT, have no way to run examples. Because they can not find any examples of TOMOATT in their PC. So, is there any solutions for this issue. For instance, provide an independent link for example downloading, or directly download examples when using conda to install TOMOATT.

**Reply by xumi1993** [reply]
I think we can create another repository for examples

**Comment by xumi1993** [comment]
Has he tried to install TomoATT manually in a conda environment?

**Comment by JingChen-Thu** [comment]
New information are provided to check the installation. The versions of Ubuntu and glibc are OK.

![d2c95e852179ca8c498bf35349d7526](https://github.com/user-attachments/assets/51ea021a-87ad-4770-a274-e767ef98551a)
